### ISSUE TITLE
executing .desktop crashes cli

### ISSUE THREAD

**Issue by ManoharOfficial** [snoopy112/FW-tool#2]
I was wondering if this tool works for the R2 Gen 2 as well? 
I would very much like to modify some default system graphics elements and a few icons here and there.
PS : I tried following your instructions, but even after installing all the dependencies, the .desktop file refuses to run.
 Using on WSL. the cli interface just crashes the moment I try to execute the .desktop file

**Comment by snoopy112** [comment]
Try to run script from terminal: **python ./FW_tool.py**
What the error do you obtain?

**Comment by ManoharOfficial** [comment]
There was no error. Just the cli of wsl crashes. However, I was able to find another method. I just found out that HiBy provides the firmware for Shanling and other players as well. AND shanling already has a published, unpacking and re-packing tool with instructions. I just unpacked the R2 ii firmware and everything seems identical/logical to Shanling theme editing steps. For my use case this is more than enough as I won't be touching any code, just GUI elements (which just seem to be images. Even the clock is just couple of images on the r2ii lol)
Here's the official link: https://www.head-fi.org/threads/shanling-daps-user-theme.866314/ 
I still have to try repacking and flashing the player tho... that is what is making me nervous lol
